providers/json.js

```diff
@@ -54,6 +55,16 @@ exports.getAll = (table) => {
 exports.get = (table, document) => fs.readJSON(resolve(baseDir, table, `${document}.json`)).catch(() => null);
 
 /**
+ * Check if the document exists.
+ * @param {string} table The name of the directory.
+ * @param {string} document The document name.
+ * @returns {Promise<boolean>}
+ */
+exports.has = (table, document) => fs.access(resolve(baseDir, table, `${document}.json`))
```

Comment:
`exports.has = (table, document) => fs.pathExists(resolve(baseDir, table, `${document}.json`));`
pathExists is exactly written the way you have it there.